colearn_grpc/proto/compile.sh

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env bash
 
-rm ./generated/*
+rm -r ./generated/*
 
 python3 -m grpc_tools.protoc \
         -I . \
         --python_out=./generated \
         --grpc_python_out=./generated \
         *.proto
 
+# protoc uses implicit relative imports which are not allowed in python3. This converts implicit imports of the
+# form "import .*_pb2" to explicit relative imports ("from . import")
+sed -i '/^import\ .*_pb2/s/^/from \. /' ./generated/*.py
```

Comment:
fancy :)

Reply:
It seems I get an error using this sed snippet : `sed: 1: "./generated/interface_p ...": invalid command code .`

Reply:
ah. it is because the mac sed is BSD (sigh). Anyway, this change works for me and hopefully works for you also `sed -i '' -e '/^import\ .*_pb2/s/^/from \. /' ./generated/*.py`